convert/jenkinsjson/json/s3upload.go

```diff
@@ -0,0 +1,103 @@
+package json
+
+import (
+	"fmt"
+
+	harness "github.com/drone/spec/dist/go"
+)
+
+func ExtractEntries(node Node) []map[string]interface{} {
+	delegate, ok := node.ParameterMap["delegate"].(map[string]interface{})
+	if !ok {
+		fmt.Println("Missing 'delegate' in parameterMap")
+		return nil
+	}
+	// Extract the 'arguments' map from the 'delegate'
+	arguments, ok := delegate["arguments"].(map[string]interface{})
+	if !ok {
+		fmt.Println("Missing 'arguments' in delegate map")
+		return nil
+	}
+	// Extract values from the "entries" in the parameterMap
+	entries, ok := arguments["entries"].([]interface{})
+	if !ok {
+		fmt.Println("No entries operations found in arguments")
+		return nil
+	}
+
+	// Convert entries to a slice of maps for easier access
+	var entryMaps []map[string]interface{}
+	for _, entry := range entries {
+		entryMap, ok := entry.(map[string]interface{})
+		if ok {
+			entryMaps = append(entryMaps, entryMap)
+		} else {
+			fmt.Println("Invalid entryMap format")
+		}
+	}
+
+	return entryMaps
+}
+
+func Converts3Upload(node Node, entryMap map[string]interface{}) *harness.Step {
+	var withPropertiesList []map[string]interface{}
+	// Check the "parameterMap" and extract "delegate"
```

Comment:
What do you mean by that comment? If not required, just remove it.

Reply:
yes not required . Removed it.